nocrypto.opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+homepage:     "https://github.com/mirleft/ocaml-nocrypto"
+dev-repo:     "git+https://github.com/mirleft/ocaml-nocrypto.git"
+bug-reports:  "https://github.com/mirleft/ocaml-nocrypto/issues"
+doc:          "https://mirleft.github.io/ocaml-nocrypto/doc"
+authors:      ["David Kaloper <dk505@cam.ac.uk>"]
+maintainer:   "David Kaloper <dk505@cam.ac.uk>"
+license:      "ISC"
+synopsis:     "Simple crypto for the modern age"
+
+build: [ ["dune" "subst"] {pinned}
+         ["dune" "build" "-p" name "-j" jobs ]
+         ["dune" "runtest"] {with-test} ]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build}
+  "cpuid" {build}
+  "ounit" {with-test}
+  "cstruct" {>="3.0.0"}
+  "ocplib-endian"
+  "zarith"
+  ("mirage-no-xen" | ("mirage-xen" & "zarith-xen"))
+  ("mirage-no-solo5" | ("mirage-solo5" & "zarith-freestanding"))
```

Comment:
are these still needed since the mirage packages are removed from this repo?

Reply:
The mirage package, `nocrypto.mirage`, is still present. It's just that it's meaningless because the stubs are not cross-compiled.

Hence the PR: the state of this branch is still incoherent.